kubeadm/flannel/versions.json

```diff
@@ -0,0 +1,22 @@
+{
+    "flannel": [
+        "0.12.0", 
+        "0.13.0"
+    ],
+    "baseimages": [
+        {
+            "dockerfile": "Dockerfile",
+            "suffix": "",
+            "base": "mcr.microsoft.com/windows/servercore",
+            "tags": ["1809"],
```

Comment:
Currently github doesn't support Windows Server 1903 or 2004 as build server.
So if we add 1903 and 2004 to tags then build will be failed because it will be run on Windows Server 1809.
```
tags: ["1809", "1903", "2004"]
```

Reply:
We need to add self-hosted runner based on Windows Server 2004 to build images for Windows Server 2004.

Reply:
:disappointed:   might need to look more seriously at buildx. It is how the test images and upstream pause image are built.

Reply:
Sorry, maybe I am so stupid, but I don't understand how to use buildx on windows. It doesn't work for me.

Reply:
your not, it doesn't.  We would have to switch to building on Linux and requires no 'RUN' commands in the windows docker images. We should be able to pull this off but probably makes sense to do in a follow up PR